python/graphstorm/inference/ep_infer.py

```diff
@@ -40,7 +40,7 @@ class GSgnnEdgePredictionInfer(GSInfer):
         The rank.
     """
 
-    def infer(self, loader, save_embed_path, save_prediction_path=None,
+    def infer(self, loader, eval_fanout, save_embed_path, save_prediction_path=None,
```

Comment:
if the data loader has fanout, why not just get the fanout from the data loader?

Reply:
Done.